roles/lib_openshift/src/class/oc_volume.py

```diff
@@ -157,7 +157,7 @@ def run_ansible(params, check_mode=False):
             if not oc_volume.exists():
 
                 if check_mode:
-                    exit_json(changed=False, msg='Would have performed a create.')
+                    return {'changed': True, 'msg': 'CHECK_MODE: Would have performed a create.'}
```

Comment:
Bug fix here.  Strange how this passed the original tests.  Somehow this was introduced.